code-challenges.js

```diff
@@ -72,17 +72,36 @@ const reversedStr = str.split("").reverse().join("");
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
 // Pseudo code:
+    // Create function named lastIndexOfArray
+    // Set parameters as array, value
+    // Create a loop that takes in all values of the array based on length, (including the last index of a repeated number)
+    // Return index of given number
 
 const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
+
+const lastIndex = (array, value) => {
+    for (let i = array.length + 1; index >= 0; index + 1) {
+        if(array[index] === value) {
+            return index;
+        }
+    }
+}
+console.log(lastIndex)
+
```

Comment:
Great, I like this approach and its very cool to see you try to use a for loop for this challenge. Couple of things to keep in mind, length will give us the number of elements in an array. Using myNumbers as an example, it would return 9 for the length and then we are saying +1  for where we want to start the loop? The increment is also + 1 but I think your intent is to start at the end and go backwards in the array. You are very close!